### PR TITLE
Resize <amp-twitter> if neither content nor fallback is available.

### PR DESCRIPTION
**Pull request by nainar**
Fix for: #13619 

This fix allows us to resize the `<amp-twitter>` component based on the placeholder element in the case where there is no content and no fallback element specified.

### CODE REVIEW

File: examples/twitter.amp.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
 <body>
 
 
-
+<!--
```
Comment (aghassemi) [inline examples/twitter.amp.html]: uncomment

File: examples/twitter.amp.html [hunk 2]
```diff
@@ -52,17 +52,22 @@ <h2>Intentionally non-existing-tweet</h2>
...
 
   <h2>Deleted tweet</h2>
-
+  <amp-twitter width=390 height=330
+      layout="fixed"
+      data-tweetid="882818033403789316"
+      data-cards="hidden">
+      <blockquote placeholder class="twitter-tweet" data-lang="en">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</blockquote>
+  </amp-twitter>
+<!--
```
Comment (aghassemi) [inline examples/twitter.amp.html]: ditto

File: extensions/amp-twitter/0.1/amp-twitter.js [hunk 3]
```diff
@@ -71,8 +71,15 @@ class AmpTwitter extends AMP.BaseElement {
       if (this.getFallback()) {
         this.togglePlaceholder(false);
         this.toggleFallback(true);
+      } else {
```
Comment (aghassemi) [inline extensions/amp-twitter/0.1/amp-twitter.js]: the issue only says placeholder, but I think we should also do it for fallback. (+ add an example with fallback to the example file).

File: examples/twitter.amp.html (hunk 1 above)
Comment (nainar) [reply examples/twitter.amp.html]: Oops done.

File: examples/twitter.amp.html (hunk 2 above)
Comment (nainar) [reply examples/twitter.amp.html]: 🤦‍♀️

File: extensions/amp-twitter/0.1/amp-twitter.js (hunk 3 above)
Comment (nainar) [reply extensions/amp-twitter/0.1/amp-twitter.js]: Done. 
1. Added the code to resize if fallback is provided
2. Added an example as well.

File: extensions/amp-twitter/0.1/amp-twitter.js [hunk 4]
```diff
@@ -68,11 +68,21 @@ class AmpTwitter extends AMP.BaseElement {
       this./*OK*/changeHeight(data['height']);
     }, /* opt_is3P */true);
     listenFor(iframe, 'no-content', () => {
-      if (this.getFallback()) {
+      const fallback = this.getFallback();
+      if (fallback) {
+        // If there is no content, but a fallback is provided.
         this.togglePlaceholder(false);
         this.toggleFallback(true);
+        this./*OK*/changeHeight(fallback.offsetHeight);
```
Comment (aghassemi) [inline extensions/amp-twitter/0.1/amp-twitter.js]: needs to become `fallback/*OK*/.offsetHeight` or presubmit will complain.

Comment (nainar) [reply extensions/amp-twitter/0.1/amp-twitter.js]: Done in latest commit.

**Pull request merged**